sdks/js/src/index.ts

```diff
@@ -694,6 +697,38 @@ export class DustAPI {
     return new Ok(r.value.tokens);
   }
 
+  async uploadFile({
```

Comment:
Unless I'm missing something, I think the name is misleading as you only create a file upload url.

Could you instead put the whole logic from `uploadFile` that you wrote in `extension/app/src/hooks/useFileUploaderService.ts` ? It would make it super easy to use by others.

Reply:
yes, that would make it definitely easier to use !